presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestSphericalGeoFunctions.java

```diff
@@ -201,6 +201,59 @@ public void testArea()
 
     private void assertArea(String wkt, double expectedArea)
     {
-        assertFunction(format("ABS(ROUND((ST_Area(to_spherical_geography(ST_GeometryFromText('%s'))) / %f - 1 ) * %d, 0))", wkt, expectedArea, 10000), DOUBLE, 0.0);
+        assertFunctionImprecise(format("ST_Area(to_spherical_geography(ST_GeometryFromText('%s')))", wkt), expectedArea, 1e-4);
     }
+
+    @Test
+    public void testLength()
```

Comment:
Could you add a test for a circular linestring?

Reply:
@seanbarker Sean, I don't see a test for a circular linestring. Could you give me a pointer?